manifest.json.template

```diff
@@ -64,13 +64,16 @@
       "explode": "true"
     },
     "org.zowe.explorer-jes": {
-      "version": "~1.0.0-STAGING"
+      "version": "~1.0.3-PR-207"
     },
     "org.zowe.explorer-mvs": {
-      "version": "~1.0.0-STAGING"
+      "version": "1.0.2"
     },
     "org.zowe.explorer-uss": {
-      "version": "~1.0.0-STAGING"
+      "version": "1.0.3"
+    },
+    "org.zowe.explorer-ui-server": {
+      "version": "~0.2.13-PR-37"
```

Comment:
We need to revert version without PR-37.

Reply:
reverted